components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts

```diff
@@ -79,4 +95,33 @@ export class ServiceGroupsRequests {
 
     return this.httpClient.get<HealthSummary>(url);
   }
+
+  public getSuggestions(type: string, text: string, filters: NodeFilter): Observable<any[]> {
+    if (text && text.length > 0) {
+      const params = this.formatFilters(filters).set('type', type).set('text', text);
+      const url = `${CONFIG_MGMT_URL}/suggestions`;
```

Comment:
I might totally be missing something here but, why do we get the suggestions from the `config-mgmt-service`?

Reply:
@afiune thanks for pointing that out! I was able to confirm the correct api: applications_url: '/apis/beta/applications', `${APPLICATIONS_URL}/services-distinct-values`